astronomer/providers/amazon/aws/sensors/emr.py

```diff
@@ -59,65 +49,18 @@ def __init__(self, *args, **kwargs) -> None:  # type: ignore[no-untyped-def]
 
 class EmrJobFlowSensorAsync(EmrJobFlowSensor):
     """
-    Async EMR Job flow sensor polls for the cluster state until it reaches
-    any of the target states.
-    If it fails the sensor errors, failing the task.
-    With the default target states, sensor waits cluster to be terminated.
-    When target_states is set to ['RUNNING', 'WAITING'] sensor waits
-    until job flow to be ready (after 'STARTING' and 'BOOTSTRAPPING' states)
-
-    :param job_flow_id: job_flow_id to check the state of cluster
-    :param target_states: the target states, sensor waits until
-        job flow reaches any of these states
-    :param failed_states: the failure states, sensor fails when
-        job flow reaches any of these states
+    This class is deprecated.
+    Please use :class: `~airflow.providers.amazon.aws.sensors.emr.EmrJobFlowSensor`.
     """
 
-    def __init__(
-        self,
-        *,
-        poll_interval: float = 5,
```

Comment:
EmrJobFlowSensor uses poke_interval not poll_interval so need to that case too I feel https://github.com/apache/airflow/blob/b260367208f9c3c09bc1da2a32abf59867ddd789/airflow/providers/amazon/aws/sensors/emr.py#L524C46-L524C46

Reply:
+1

Reply:
indeed, it might be missed in https://github.com/astronomer/astronomer-providers/pull/640. let me add a similar deprecation here.